.travis.yml

```diff
@@ -56,7 +56,8 @@ jobs:
         - TEST="compile"
       before_install:
         - sudo apt-get update
-        - sudo apt-get -q -y install bison flex libjson-c-dev autoconf-archive libxen-dev
+        - sudo apt-get -q -y install bison flex libjson-c-dev autoconf-archive libxen-dev libstdc++-10-dev
+        - "sudo sed -i 's/private __detail::__variant::_Variant_base<_Types...>/public __detail::__variant::_Variant_base<_Types...>/' /usr/include/c++/10/variant"
```

Comment:
This is the moment when I feel like:
https://twitter.com/tklengyel/status/1304134567822069767

I have no clue what is the problem, this doesn't build only on Travis, there is no such error when the package is built on `ubuntu:focal` Docker image. I don't know what would be the proper solution here (other than waiting till they fix this obvious bug upstream and the patch is delivered to the distribution)